### ISSUE TITLE
[Minor] New Relic plugin errors

### ISSUE THREAD

**Issue by mjbrender** [drewkerrigan/riak-inverted-index-demo#11]
While running the web server, New Relic seems it wants to be configured: 

``` 
[06/16/15 10:29:30 -0400 Basho (84141)] ERROR : Visit NewRelic.com to obtain a valid license key, or to upgrade your account.
```

It'd be helpful to document how to set it up for this demo.

**Comment by drewkerrigan** [comment]
Yes, In fact I think NewRelic shouldn't be required for basic demo purposes. The only reason NR is involved is because this was at one time a production app; For basic public consumption, this should be made optional.
